tests/data/powerwalker-vfi_500r1u.json

```diff
@@ -0,0 +1,717 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.3808.1.1.1",
+                    "sysDescr": "Power Management Card",
+                    "sysContact": "<private>",
+                    "version": "1.0.6",
+                    "hardware": null,
```

Comment:
There is no hardware Oid available?

Reply:
SNMPv2-SMI::mib-2.1.1.0 = STRING: "Power Management Card"
SNMPv2-SMI::mib-2.1.2.0 = OID: SNMPv2-SMI::enterprises.3808.1.1.1
SNMPv2-SMI::mib-2.1.3.0 = Timeticks: (1735568600) 200 days, 21:01:26.00
SNMPv2-SMI::mib-2.1.4.0 = STRING:
SNMPv2-SMI::mib-2.1.5.0 = STRING:
SNMPv2-SMI::mib-2.1.6.0 = STRING:
SNMPv2-SMI::mib-2.1.7.0 = INTEGER: 72
SNMPv2-SMI::mib-2.2.1.0 = INTEGER: 2
SNMPv2-SMI::mib-2.2.2.1.1.1 = INTEGER: 1
SNMPv2-SMI::mib-2.2.2.1.1.2 = INTEGER: 2
SNMPv2-SMI::mib-2.2.2.1.2.1 = Hex-STRING: 61 67 01 00 00
SNMPv2-SMI::mib-2.2.2.1.2.2 = Hex-STRING: 00 08 35 12 00
SNMPv2-SMI::mib-2.2.2.1.3.1 = INTEGER: 0
SNMPv2-SMI::mib-2.2.2.1.3.2 = INTEGER: 18
SNMPv2-SMI::mib-2.2.2.1.4.1 = INTEGER: 1500
SNMPv2-SMI::mib-2.2.2.1.4.2 = INTEGER: 4649
SNMPv2-SMI::mib-2.2.2.1.5.1 = Gauge32: 10000000
SNMPv2-SMI::mib-2.2.2.1.5.2 = Gauge32: 134222393
SNMPv2-SMI::mib-2.2.2.1.6.1 = Hex-STRING: 00 0C 15 03 51 2B
SNMPv2-SMI::mib-2.2.2.1.6.2 = ""
SNMPv2-SMI::mib-2.2.2.1.7.1 = INTEGER: 7
SNMPv2-SMI::mib-2.2.2.1.7.2 = INTEGER: 2
SNMPv2-SMI::mib-2.2.2.1.8.1 = INTEGER: 1
SNMPv2-SMI::mib-2.2.2.1.8.2 = INTEGER: 2
SNMPv2-SMI::mib-2.2.2.1.9.1 = Timeticks: (8343900) 23:10:39.00
SNMPv2-SMI::mib-2.2.2.1.9.2 = Timeticks: (134222397) 15 days, 12:50:23.97
SNMPv2-SMI::mib-2.2.2.1.10.1 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.10.2 = Counter32: 134222401
SNMPv2-SMI::mib-2.2.2.1.11.1 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.11.2 = Counter32: 134222405
SNMPv2-SMI::mib-2.2.2.1.12.1 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.12.2 = Counter32: 134222409
SNMPv2-SMI::mib-2.2.2.1.13.1 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.13.2 = Counter32: 134222413
SNMPv2-SMI::mib-2.2.2.1.14.1 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.14.2 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.15.1 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.15.2 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.16.1 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.16.2 = Counter32: 134222417
SNMPv2-SMI::mib-2.2.2.1.17.1 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.17.2 = Counter32: 134222421
SNMPv2-SMI::mib-2.2.2.1.18.1 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.18.2 = Counter32: 134222425
SNMPv2-SMI::mib-2.2.2.1.19.1 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.19.2 = Counter32: 134222429
SNMPv2-SMI::mib-2.2.2.1.20.1 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.20.2 = Counter32: 0
SNMPv2-SMI::mib-2.2.2.1.21.1 = Gauge32: 0
SNMPv2-SMI::mib-2.2.2.1.21.2 = Gauge32: 0
SNMPv2-SMI::mib-2.2.2.1.22.1 = OID: SNMPv2-SMI::zeroDotZero
SNMPv2-SMI::mib-2.2.2.1.22.2 = OID: SNMPv2-SMI::zeroDotZero
SNMPv2-SMI::mib-2.4.1.0 = INTEGER: 2
SNMPv2-SMI::mib-2.4.2.0 = INTEGER: 255
SNMPv2-SMI::mib-2.4.3.0 = Counter32: 5149778
SNMPv2-SMI::mib-2.4.4.0 = Counter32: 0
SNMPv2-SMI::mib-2.4.5.0 = Counter32: 0
SNMPv2-SMI::mib-2.4.6.0 = Counter32: 0
SNMPv2-SMI::mib-2.4.7.0 = Counter32: 0
SNMPv2-SMI::mib-2.4.8.0 = Counter32: 0
SNMPv2-SMI::mib-2.4.9.0 = Counter32: 5149784
SNMPv2-SMI::mib-2.4.10.0 = Counter32: 5149733
SNMPv2-SMI::mib-2.4.11.0 = Counter32: 0
SNMPv2-SMI::mib-2.4.12.0 = Counter32: 11
SNMPv2-SMI::mib-2.4.13.0 = INTEGER: 0
SNMPv2-SMI::mib-2.4.14.0 = Counter32: 0
SNMPv2-SMI::mib-2.4.15.0 = Counter32: 0
SNMPv2-SMI::mib-2.4.16.0 = Counter32: 0
SNMPv2-SMI::mib-2.4.17.0 = Counter32: 0
SNMPv2-SMI::mib-2.4.18.0 = Counter32: 0
SNMPv2-SMI::mib-2.4.19.0 = Counter32: 0
SNMPv2-SMI::mib-2.4.23.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.1.0 = Counter32: 123331
SNMPv2-SMI::mib-2.5.2.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.3.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.4.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.5.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.6.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.7.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.8.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.9.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.10.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.11.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.12.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.13.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.14.0 = Counter32: 123331
SNMPv2-SMI::mib-2.5.15.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.16.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.17.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.18.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.19.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.20.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.21.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.22.0 = Counter32: 123331
SNMPv2-SMI::mib-2.5.23.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.24.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.25.0 = Counter32: 0
SNMPv2-SMI::mib-2.5.26.0 = Counter32: 0
SNMPv2-SMI::mib-2.6.1.0 = INTEGER: 4
SNMPv2-SMI::mib-2.6.2.0 = INTEGER: 1000
SNMPv2-SMI::mib-2.6.3.0 = INTEGER: 60000
SNMPv2-SMI::mib-2.6.4.0 = INTEGER: 60
SNMPv2-SMI::mib-2.6.5.0 = Counter32: 0
SNMPv2-SMI::mib-2.6.6.0 = Counter32: 287
SNMPv2-SMI::mib-2.6.7.0 = Counter32: 0
SNMPv2-SMI::mib-2.6.8.0 = Counter32: 280
SNMPv2-SMI::mib-2.6.9.0 = Gauge32: 0
SNMPv2-SMI::mib-2.6.10.0 = Counter32: 2062
SNMPv2-SMI::mib-2.6.11.0 = Counter32: 1709
SNMPv2-SMI::mib-2.6.12.0 = Counter32: 0
SNMPv2-SMI::mib-2.6.13.7.1.0 = Counter32: 5024487
SNMPv2-SMI::mib-2.6.14.0 = Counter32: 0
SNMPv2-SMI::mib-2.6.15.0 = Counter32: 41
SNMPv2-SMI::mib-2.7.1.0 = Counter32: 5024490
SNMPv2-SMI::mib-2.7.2.0 = Counter32: 0
SNMPv2-SMI::mib-2.7.3.0 = Counter32: 0
SNMPv2-SMI::mib-2.7.4.0 = Counter32: 5024511
SNMPv2-SMI::mib-2.7.5.1.1.0.0.0.0.53 = IpAddress: 0.0.0.0
SNMPv2-SMI::mib-2.7.5.1.1.0.0.0.0.161 = IpAddress: 0.0.0.0
SNMPv2-SMI::mib-2.7.5.1.1.0.0.0.0.53565 = IpAddress: 0.0.0.0
SNMPv2-SMI::mib-2.7.5.1.1.0.0.0.0.53566 = IpAddress: 0.0.0.0
SNMPv2-SMI::mib-2.7.5.1.2.0.0.0.0.53 = INTEGER: 53
SNMPv2-SMI::mib-2.7.5.1.2.0.0.0.0.161 = INTEGER: 161
SNMPv2-SMI::mib-2.7.5.1.2.0.0.0.0.53565 = INTEGER: 53565
SNMPv2-SMI::mib-2.7.5.1.2.0.0.0.0.53566 = INTEGER: 53566
SNMPv2-SMI::mib-2.11.1.0 = Counter32: 5024401
SNMPv2-SMI::mib-2.11.2.0 = Counter32: 5024401
SNMPv2-SMI::mib-2.11.3.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.4.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.5.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.6.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.8.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.9.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.10.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.11.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.12.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.13.0 = Counter32: 7751922
SNMPv2-SMI::mib-2.11.14.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.15.0 = Counter32: 1303484
SNMPv2-SMI::mib-2.11.16.0 = Counter32: 3720931
SNMPv2-SMI::mib-2.11.17.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.18.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.19.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.20.0 = Counter32: 57425
SNMPv2-SMI::mib-2.11.21.0 = Counter32: 783204
SNMPv2-SMI::mib-2.11.22.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.24.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.25.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.26.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.27.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.28.0 = Counter32: 5024425
SNMPv2-SMI::mib-2.11.29.0 = Counter32: 0
SNMPv2-SMI::mib-2.11.30.0 = INTEGER: 2
UPS-MIB::upsIdentManufacturer.0 = STRING: CPS
UPS-MIB::upsIdentModel.0 = STRING: 500R
UPS-MIB::upsIdentUPSSoftwareVersion.0 = STRING: 1.0.6
UPS-MIB::upsIdentName.0 = STRING: PowerWalker
UPS-MIB::upsBatteryStatus.0 = INTEGER: batteryNormal(2)
UPS-MIB::upsSecondsOnBattery.0 = INTEGER: 0 seconds
UPS-MIB::upsEstimatedMinutesRemaining.0 = INTEGER: 21 minutes
UPS-MIB::upsEstimatedChargeRemaining.0 = INTEGER: 100 percent
UPS-MIB::upsBatteryVoltage.0 = INTEGER: 136 0.1 Volt DC
UPS-MIB::upsBatteryTemperature.0 = Wrong Type (should be INTEGER): NULL
UPS-MIB::upsInputNumLines.0 = INTEGER: 1
UPS-MIB::upsInputLineIndex.1 = INTEGER: 1
UPS-MIB::upsInputFrequency.1 = INTEGER: 501 0.1 Hertz
UPS-MIB::upsInputVoltage.1 = INTEGER: 227 RMS Volts
UPS-MIB::upsInputCurrent.1 = INTEGER: 0 0.1 RMS Amp
UPS-MIB::upsInputTruePower.1 = INTEGER: 0 Watts
UPS-MIB::upsOutputSource.0 = INTEGER: normal(3)
UPS-MIB::upsOutputFrequency.0 = INTEGER: 500 0.1 Hertz
UPS-MIB::upsOutputNumLines.0 = INTEGER: 1
UPS-MIB::upsOutputLineIndex.1 = INTEGER: 1
UPS-MIB::upsOutputVoltage.1 = INTEGER: 225 RMS Volts
UPS-MIB::upsOutputCurrent.1 = INTEGER: 0 0.1 RMS Amp
UPS-MIB::upsOutputPower.1 = INTEGER: 0 Watts
UPS-MIB::upsOutputPercentLoad.1 = INTEGER: 28 percent
UPS-MIB::upsBypassFrequency.0 = INTEGER: 501 0.1 Hertz
UPS-MIB::upsBypassNumLines.0 = INTEGER: 1
UPS-MIB::upsAlarmsPresent.0 = Gauge32: 0
UPS-MIB::upsTestId.0 = OID: UPS-MIB::upsTestQuickBatteryTest
UPS-MIB::upsTestSpinLock.0 = INTEGER: 7
UPS-MIB::upsTestResultsSummary.0 = INTEGER: donePass(1)
UPS-MIB::upsTestResultsDetail.0 = STRING:
UPS-MIB::upsTestStartTime.0 = Timeticks: (1711080500) 198 days, 1:00:05.00
Timeout: No Response from #.#.#.#

Reply:
Second most important thing is that need to be excluded from Cyberpower OS:
![image](https://user-images.githubusercontent.com/67277751/118615732-4fc7d700-b7c1-11eb-998b-fd5e03033132.png)

Reply:
I have a Phasak UPS that uses the same SNMP Card.

It's detected as a PowerWalker UPS and that leads to not extracting the proper values.

When it's detected as cyberpower it reports the correct values.

Maybe the         sysDescr_except:
            - 'Power Management Card' 
should be improved?